src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Program.cs

```diff
@@ -22,11 +23,34 @@
 
 builder.Services.AddTransient<ISftpService, SftpService>();
 
+var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddConsole();
+});
+
+ILogger logger = loggerFactory.CreateLogger<Startup>();
+
 builder.Services.AddTransient<SftpClient>(services =>
 {
     var configurationOptions = services.GetRequiredService<IOptions<SftpConfig>>();
     var configuration = configurationOptions.Value;
-    return new SftpClient(configuration.Host, configuration.Port == 0 ? 22 : configuration.Port, configuration.Username, configuration.Password);
+    try
+    {
+        var privateKey = new PrivateKeyFile(configuration.SshPrivateKeyPath);
```

Comment:
@ChrisHoban when we deploy to OpenShift, should we prefer loading the ssh private key from a environment variable?  the library we are using can read from a `Stream` which is easy to load from an environment variable.